_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -0,0 +1,599 @@
+						</a>
+					</h3>
+				</div>
+				<div class="desc column-description">
+					<p><?php echo esc_html( $description ); ?></p>
+				</div>
+
+				<div class="details-link">
+					<a class="jptracks thickbox open-plugin-details-modal" href="<?php echo esc_url( $details_link ); ?>" data-jptracks-name="jetpack_about_plugin_details_modal" data-jptracks-prop="<?php echo esc_attr( $plugin['slug'] ); ?>"><?php esc_html_e( 'More Details', 'jetpack' ); ?></a>
+				</div>
+			</div>
+
+			<div class="plugin-card-bottom">
+				<div class="meta">
+					<?php
+					wp_star_rating(
+						array(
+							'rating' => $plugin['rating'],
+							'type'   => 'percent',
+							'number' => $plugin['num_ratings'],
+						)
+					);
+					?>
+					<span class="num-ratings" aria-hidden="true">(<?php echo esc_html( number_format_i18n( $plugin['num_ratings'] ) ); ?> <?php esc_html_e( 'ratings', 'jetpack' ); ?>)</span>
+					<div class="downloaded">
+						<?php
+						if ( $plugin['active_installs'] >= 1000000 ) {
+							$active_installs_millions = floor( $plugin['active_installs'] / 1000000 );
+							$active_installs_text     = sprintf(
+								/* translators: number of millions of installs. */
+								_nx( '%s+ Million', '%s+ Million', $active_installs_millions, 'Active plugin installations', 'jetpack' ),
+								number_format_i18n( $active_installs_millions )
+							);
+						} elseif ( 0 === $plugin['active_installs'] ) {
+							$active_installs_text = _x( 'Less Than 10', 'Active plugin installations', 'jetpack' );
+						} else {
+							$active_installs_text = number_format_i18n( $plugin['active_installs'] ) . '+';
+						}
+						/* translators: number of active installs */
+						printf( esc_html__( '%s Active Installations', 'jetpack' ), esc_html( $active_installs_text ) );
+						?>
+					</div>
+				</div>
+
+				<div class="action-links">
+					<?php
+					if ( $action_links ) {
+						// The var simply collects strings that have already been sanitized.
+						// phpcs:ignore WordPress.Security.EscapeOutput
+						echo '<ul class="action-buttons"><li>' . implode( '</li><li>', $action_links ) . '</li></ul>';
+					}
+					?>
+				</div>
+			</div>
+		</li>
+			<?php
+
+		}
+
+	}
+
+	/**
+	 * Fetch Gravatar hashes for public A12s from wpcom and display them as a list.
+	 */
+	public function display_gravatars() {
+		$hashes = get_transient( 'a12s_hashes' );
+		if ( false === $hashes ) {
+			$response = json_decode(
+				wp_remote_retrieve_body(
+					wp_remote_get( 'https://public-api.wordpress.com/wpcom/v2/a11n-gravatar-hashes' )
+				)
+			);
+			if ( ! empty( $response ) && is_array( $response ) ) {
+				$hashes = array();
+				foreach ( $response as $hash ) {
+					$hashes[] = "https://2.gravatar.com/avatar/$hash";
+				}
+				if ( ! empty( $hashes ) ) {
+					set_transient( 'a12s_hashes', $hashes, DAY_IN_SECONDS );
+				} else {
+					$hashes = array(
+						'https://2.gravatar.com/avatar/5ef318426c941cbef6db5342c1356231',
```

Comment:
What do you think about adding a bit of diversity in that default list? Right now it's 16 white males. While that list represented a8c in its first few years, I think we have a more diverse group of people now.

Reply:
Nice. Done.